tcpdf.php

```diff
@@ -16396,8 +16396,8 @@ protected function getHtmlDomArray($html) {
 		// create a special tag to contain the CSS array (used for table content)
 		$csstagarray = '<cssarray>'.htmlentities(json_encode($css)).'</cssarray>';
 		// remove head and style blocks
-		$html = preg_replace('/<head([^\>]*)>(.*)?<\/head>/siU', '', $html);
-		$html = preg_replace('/<style([^\>]*)>([^\<]*)<\/style>/isU', '', $html);
+		$html = preg_replace('/<head([^\>]*?)>(.*)<\/head>/is', '', $html);
```

Comment:
```suggestion
		$html = preg_replace('/<head([^\>]*?)>(.*?)<\/head>/is', '', $html);
```
this works, right? Please also provide a link to full test file.

Reply:
yes, this works

example mini test project:  [test-tcpdf.zip](https://github.com/tecnickcom/TCPDF/files/6271174/test-tcpdf.zip)
Unpack, then run:

```
composer up
php ./test.php
```